File: Topic Wise Apps/Notification_Example/app/src/main/java/com/example/myapp/MainActivity.java
package com.example.myapp;

import androidx.appcompat.app.AppCompatActivity;
import androidx.core.app.NotificationCompat;

import android.app.NotificationChannel;
import android.app.NotificationManager;
import android.app.PendingIntent;
import android.content.Intent;
import android.graphics.Color;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {
NotificationManager nm;
PendingIntent pendingIntent;
Intent intent;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        nm= (NotificationManager) getSystemService(NOTIFICATION_SERVICE);
        creaChannel();
        intent=new Intent(this, MainActivity.class);
        pendingIntent=PendingIntent.getActivity(this,2234,intent,PendingIntent.FLAG_UPDATE_CURRENT);

    }

    private void creaChannel() {
        if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.O) {
            NotificationChannel notificationChannel=new NotificationChannel("MyId","muni",NotificationManager.IMPORTANCE_HIGH);
                notificationChannel.setLightColor(Color.GREEN);
                nm.createNotificationChannel(notificationChannel);

        }

    }

    public void showNotifi(View view) {
        NotificationCompat.Builder builder=new NotificationCompat.Builder(this,"MyId");
        builder.setContentText("Example");
        builder.setContentTitle("Test");
        builder.setContentIntent(pendingIntent);
        builder.setSmallIcon(R.drawable.ic_launcher_foreground);
        nm.notify(2333,builder.build());

    }
}

File: MusicPlayer/app/src/main/java/com/example/musicplayer/MainActivity.kt
package com.example.musicplayer

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.ImageView

class MainActivity : AppCompatActivity() {
    lateinit var imageView: ImageView
    var btn=true
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    imageView=findViewById(R.id.imageView)
        imageView.setOnClickListener(View.OnClickListener {

            if (btn==true){
                imageView.setImageResource(R.drawable.stop)
                val intent=Intent(this,
                    MyMusicService::class.java)
                startService(intent)
                btn=false

            }else{
                imageView.setImageResource(R.drawable.play)
                val intent=Intent(this,
                    MyMusicService::class.java)
                stopService(intent)
                btn=true
            }
        })
    }
}